browser/src/map/handler/Map.Keyboard.js

```diff
@@ -477,17 +578,17 @@ L.Map.Keyboard = L.Handler.extend({
 			return;
 
 		// Control
-		if (e.keyCode == 17)
+		if (e.keyCode == this.keyCodes.CTRL)
 			return true;
 
 		if (e.type !== 'keydown' && e.key !== 'c' && e.key !== 'v' && e.key !== 'x' &&
-		/* Safari */ e.keyCode !== 99 && e.keyCode !== 118 && e.keyCode !== 120) {
+		/* Safari */ e.keyCode !== this.keyCodes.NUM3[1] && e.keyCode !== this.keyCodes.F7 && e.keyCode !== this.keyCodes.F9) {
```

Comment:
hmm, it shouldn't be misleading:
`keyCodes.NUM3[1]` -> `keyCodes.c`  - as we already have dedicated name for it

Reply:
How did you know `99` here refers to `c`? Was it by looking at the if statement above it? If Yes then I guess I need to change `118` and `120` too.